infrastructure/aat.tfvars

```diff
@@ -8,4 +8,5 @@ sya_hostname  = "benefit-appeal.nonprod.platform.hmcts.net"
 
 idam_api_url = "https://idam-api.aat.platform.hmcts.net"
 idam_login_url = "https://idam-web-public.aat.platform.hmcts.net/register"
+postcode_lookup_enabled= "true"
```

Comment:
you need to do the same in the prod.tvars file

Reply:
will do after preview test completed